apps/tangle-dapp/src/components/account/PromotionalBanner.tsx

```diff
@@ -0,0 +1,141 @@
+import { FC, useState, useEffect } from 'react';
+import { Card, Typography } from '@webb-tools/webb-ui-components';
+import { twMerge } from 'tailwind-merge';
+import { ArrowRightUp } from '@webb-tools/icons';
+
+interface CarouselSlide {
+  title: string;
+  description: string;
+  bulletPoints: string[];
+  btnText: string;
+  btnLink: string;
+}
+
+const slides: CarouselSlide[] = [
```

Comment:
To maintain consistency with the other codebase for constants:

```suggestion
const SLIDES: CarouselSlide[] = [
```